.github/workflows/update-base.yml

```
@@ -0,0 +1,76 @@
+name: Update base image base
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every Monday at midnight
+    - cron: "0 0 * * 1"
```

```suggestion
    - cron: "0 0 * * 4"
```

If we do on Thursday, it'll give the PR time to run checks. This way, the Engine deployer could potentially merge Friday.

Why is this timeline helpful? We'd then produce a Packer Image on Monday.

More timeline info for shipping workspace clusters here: https://www.notion.so/gitpod/Generation-eb8b655c6aa94373a73896c8055f5c9a

---

Sounds good, thanks!